### ISSUE TITLE
[Python] Response composition in Python is missing some import

### ISSUE THREAD

**Issue by guyou** [swagger-api/swagger-codegen#10414]
##### Description

When using message composition, the Python generated code is missing some import.

##### Swagger-codegen version

3.0.20

##### Swagger declaration file content or url

The specification contains:

```yaml
components:
  schemas:
    simple_response:
      title: simple response
      type: object
      required:
      - return
      properties:
        return:
          type: integer
          description: indicateur de succès de l'opération
          enum:
            - 0
            - 1
    error_response:
      title: error message
      allOf:
      - $ref: '#/components/schemas/simple_response'
      - type: object
        required:
        - msg
        properties:
          msg:
            type: string
            description: le message d'erreur
```

##### Command line used for generation

    java -jar swagger-codegen-cli.jar generate -i openkarotz-openapi.json -l python -o python

##### Steps to reproduce

When running the generated code, the python generates the following error:
```
Traceback (most recent call last):
  File "./test-python.py", line 3, in <module>
    import pyopenkarotz
  File "/home/user/Devel/openkarotz-openapi/python/pyopenkarotz/__init__.py", line 18, in <module>
    from pyopenkarotz.api.default_api import DefaultApi
  File "/home/user/Devel/openkarotz-openapi/python/pyopenkarotz/api/__init__.py", line 6, in <module>
    from pyopenkarotz.api.default_api import DefaultApi
  File "/home/user/Devel/openkarotz-openapi/python/pyopenkarotz/api/default_api.py", line 20, in <module>
    from pyopenkarotz.api_client import ApiClient
  File "/home/user/Devel/openkarotz-openapi/python/pyopenkarotz/api_client.py", line 26, in <module>
    import pyopenkarotz.models
  File "/home/user/Devel/openkarotz-openapi/python/pyopenkarotz/models/__init__.py", line 24, in <module>
    from pyopenkarotz.models.error_response import ErrorResponse
  File "/home/user/Devel/openkarotz-openapi/python/pyopenkarotz/models/error_response.py", line 19, in <module>
    class ErrorResponse(object):
  File "/home/user/Devel/openkarotz-openapi/python/pyopenkarotz/models/error_response.py", line 35, in ErrorResponse
    if hasattr(SimpleResponse, "swagger_types"):
NameError: name 'SimpleResponse' is not defined
``

##### Related issues/PRs

<!-- has a similar issue/PR been reported/opened before? Please do a search in https://github.com/swagger-api/swagger-codegen/issues?utf8=%E2%9C%93&q=is%3Aissue%20 -->

##### Suggest a fix/enhancement

It seems an import is missing in the generated class for model, when depending on an other model.

**Comment by guyou** [comment]
Seems related to the `handlebars/python/model.mustache` file. Should I open an issue to https://github.com/swagger-api/swagger-codegen-generators?

**Comment by guyou** [comment]
Fixed by https://github.com/swagger-api/swagger-codegen-generators/commit/53394a0597f85c710b5bc37cdfb6911c060face1
Released in swagger-codegen-generators v1.0.21.
Integrated in swagger-codegen-cli v3.0.21 (published yesterday, just unlucky).
